pkg/clustermesh/operator/remote_cluster.go

```diff
@@ -97,13 +126,32 @@ func (s *synced) Services(ctx context.Context) error {
 	return s.Wait(ctx, s.services.WaitChannel())
 }
 
+// ServiceExports returns after that the initial list of shared MCS-API
+// service exports has been received from the remote cluster, the remote
+// cluster is disconnected, or the given context is canceled.
+func (s *synced) ServiceExports(ctx context.Context) error {
+	return s.Wait(ctx, s.serviceExports.WaitChannel())
+}
```

Comment:
Depending on the expectations of the consumer, you may want to additionally mark the services as synced if disabled.

Reply:
Ahh I think I will just remove this for now, I didn't noticed it depends on code that I don't want to write in this PR (something a bit similar to GlobalServiceCache but for service exports)